source/common/ssl/utility.cc

```diff
@@ -8,6 +8,13 @@
 namespace Envoy {
 namespace Ssl {
 
+inline bssl::UniquePtr<ASN1_TIME> epochASN1_Time() {
```

Comment:
use `CONSTRUCT_ON_FIRST_USE`? This doesn't need to be calculated every time.

Reply:
@lizan I tried to use that and I get the following erro 
copy constructor is implicitly deleted because` 'unique_ptr<asn1_string_st, bssl::internal::Deleter<asn1_string_st> >`' has a user-declared move constructor. any way to get around this?

Reply:
In that case you don't need use UniquePtr since they only construct once and live the whole process liftime, ASN1_TIME is enough.

Reply:
@lizan ok. Cool. Thanks. I have updated. PTAL.